flixel/system/FlxSound.hx

```diff
@@ -144,6 +144,10 @@ class FlxSound extends FlxBasic
 	 * Helper var to prevent the sound from playing after focus was regained when it was already paused.
 	 */
 	private var _alreadyPaused:Bool = false;
+	/**
+	 * In case of looping, from where to restart the sound when it loops back
+	 */
+	private var loopPoint(default, set):Float;
```

Comment:
I'm also not sure I like the name... How about `loopTime` (there already is a `time`) or `loopPosition`?